packages/truss/src/sections/tachyons/widths.ts

```diff
@@ -38,6 +41,9 @@ export const width: CreateMethodsFn = (config) => [
       maxw50: "50%",
       maxw75: "75%",
       maxw100: "100%",
+      maxwfc: "fit-content",
+      maxwmin: "min-content",
```

Comment:
Should this be maxwminc ( with c at the end)?

Reply:
Ah yeah! Good catch!